docs/fastdds/library_overview/includes/exec_model.rst

```diff
@@ -15,14 +15,69 @@ any methods on the same DomainParticipant from different threads.
 However, this multithreading implementation must be taken into account when external functions access to resources that
 are modified by threads running internally in the library.
 An example of this is the modified resources in the entity listener callbacks.
-The following is a brief overview of how *Fast DDS* multithreading schedule work:
-
-* Main thread: Managed by the application.
-* Event thread: Each DomainParticipant owns one of these. It processes periodic and triggered time events.
-* Asynchronous writer thread: This thread manages asynchronous writes for all DomainParticipants.
-  Even for synchronous writers, some forms of communication must be initiated in the background.
-* Reception threads: DomainParticipants spawn a thread for each reception channel, where the concept of a channel
-  depends on the transport layer (e.g. a UDP port).
+
+The complete set of threads spawned by Fast DDS is shown below.
+Transport related threads (marked as UDP, TCP and SHM types) are only created when the appropriate Transport is used.
+
+.. list-table::
+    :header-rows: 1
+    :align: left
+
+    * - Name
+      - Type
+      - Cardinality
+      - Description
+    * - Event
+      - General
+      - One per DomainParticipant
+      - Processes periodic and triggered time events
+    * - Asynchronous Writer
+      - General
+      - One per enabled asynchronous flow controller
+      - Manages asynchronous writes.
+
+        Even for synchronous writers, some forms of communication must be initiated in the background.
+    * - Datasharing Listener
+      - General
+      - One per DomainParticipant
+      - Listener thread that processes messages received via Datasharing
+    * - Reception
+      - UDP
+      - One per port
```

Comment:
Either we link to the default ports somehow (IDK whether we have that) or we elabore further here on the number of ports

Reply:
I don't think this has been addressed

Reply:
Nevermind